code/switch.js

```diff
@@ -0,0 +1,195 @@
+if (userName === "") {
+  alert("Sorry, didn't catch your name. Please enter your name again.");
+  // The process.exit(1) command is used to exit a Node.js program with a non-zero exit code.
+  // It immediately ends the program's execution and indicates that something went wrong or an error occurred.
+  process.exit(1);
+} else {
+  // Greeting which uses the user's entered name.
+  alert(
+    `Hi ${userName}! Welcome to the Javascript Pizzeria!`);
+}
+
+// Step 2 - Food choice
+
+// Variables for food categories and corresponding user choice
+let foodCategory = "";  //pizza, pinsa, dessert pizza
+let selectedFoodCategory = "";
+
+// Prompt asking the user to enter a number related to the food category they wanna choose.
+foodCategory = prompt(`What would you like to order today? 
+    Please enter the number of your choice:
+  1 - Pizza
+  2 - Pinsa
+  3 - Dessert Pizza`);
+
+// Switch-statement to relate the different food categories with the user's choice. An error message will appear if they choose something else. I am thinking of storing the error message in its own variable (since they look very much the same with small variations) but I will leave this as it is for the moment since I'm not sure what is better/easier to read.
+switch (foodCategory) {
+  case "1":
+    selectedFoodCategory = "Pizza";
+    break;
+  case "2":
+    selectedFoodCategory = "Pinsa";
+    break;
+  case "3":
+    selectedFoodCategory = "Dessert Pizza";
+    break;
+  default:
+    alert("Invalid menu choice. Please enter (1), (2) or (3).");
+    process.exit(1);
+}
+
+// Message and confirmation to the user about their selected food category.
+alert(`You've chosen ${selectedFoodCategory}! What a great choice ${userName}!`);
+
+// Initializing variables related to menu choices for the different food categories and corresponding prices for children and adult size pizzas respectively (string and arrays).
+let foodSubType = "";
+let foodOption = [];
+let foodPrice = [];
+
+// Switch-statement that relates the different food categories with a prompt to the user for each category, giving them menu alternatives to choose from.
+// For each food category there's an array with the menu choices as well as the corresponding prices.
+// In later iterations I wanna try storing all data further up in the code. If someone adds items to this code I would have update multiple places which isn't ideal. A better solution I think would be to not have the arrays in this part of the code and use a variable here instead.
+switch (selectedFoodCategory) {
+  case "Pizza":
+    foodSubType = prompt(`Select the pizza that you wanna order:
+        1 - Pizza Margherita
+        2 - Pizza Funghi 
+        3 - Pizza Quattro Formaggi
+        Please enter the number of your choice.`);
+    foodOption = ["Pizza Margherita", "Pizza Funghi", "Pizza Quattro Formaggi"];  // Pizza choices
+    foodPrice = [
+      ["70", "80", "75"],   // Price for adult size, sek
```

Comment:
I'm impressed with this array with the different food prices for each and every food choice, cool!